EIPS/eip-1.md

```diff
@@ -3,121 +3,108 @@ eip: 1
 title: EIP Purpose and Guidelines
 status: Active
 type: Meta
-author: Martin Becze <mb@ethereum.org>, Hudson Jameson <hudson@ethereum.org>
+author: Martin Becze <mb@ethereum.org>, Hudson Jameson <hudson@ethereum.org>, William Entriken <github.com@phor.net>
 created: 2015-10-27, 2017-02-01
 ---
 
-What is an EIP?
---------------
+## What is an EIP?
```

Comment:
What's the reasoning behind changing all the header styles?

Reply:
I use Typora editor, this is how the editor is saving the file.

I have reviewed every FINAL status EIP and they are all using the format that this PR uses.

Reply:
It's also easier to prepend a heading with a couple of hashes than it is to use dashes on the next line (and line them up for presentation, and have to update them each time the heading is changed).

Reply:
If you want to reformat EIP 1, please submit a separate PR for it. It's very hard to find the content changes amongst all the formatting changes here.